hack/components/bump-kubemacpool.sh

```diff
@@ -46,7 +47,18 @@ spec:
         imagePullPolicy: "{{ .ImagePullPolicy }}"
         name: manager
 EOF
+
+    cat <<EOF > config/cnao/cnao_rejectowner_patch.yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: service
+  namespace: system
+  annotations:
+    networkaddonsoperator.network.kubevirt.io/rejectOwner: ""
```

Comment:
Can we add this annotation only at TLS secret ? 
https://github.com/k8snetworkplumbingwg/kubemacpool/blob/master/config/default/manager/manager.yaml#L201

Reply:
That's what we do, no? The Secret is identified by namespace and name. We can't have two secrets with the same name, so this should work ok.

Reply:
Ahh you right !